second lesson

AnnaSolovykh opened the pull request:
Lesson 1-2 Hometask

Jkuffler commented:
Overall - this works! I just can't seem to find which line of code has the additional semicolon `;` that's being rendered to the bottom of the list... Perhaps you will find it before I do :)

AnnaSolovykh commented:
Thank you a lot for such a considerate way of showing the mistake! I really appreciate it!

I have deleted the unneeded semicolon - and added some new to make the code cleaner.

Jkuffler commented:
Awesome!
